dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Grid/FullDynamicModel.mo

```diff
@@ -169,84 +169,103 @@ model FullDynamicModel "Nordic test grid with buses, lines, shunts, voltage-depe
 protected
   // Generator g01 init values:
   // P0Pu, Q0Pu in SnRef, receptor convention
-  final parameter Types.PerUnit P0Pu_g01 = -600.00 / Electrical.SystemBase.SnRef;
-  final parameter Types.PerUnit Q0Pu_g01 = -58.3 / Electrical.SystemBase.SnRef;
+  final parameter Types.ActivePowerPu P0Pu_g01 = -6;
+  final parameter Types.ReactivePowerPu Q0Pu_g01 = -0.583;
```

Comment:
The Q0Pu here need to be taken from the load flow and updated for case B, so maybe not a final parameter ?